test/lower-non-standard-types.ll

```diff
@@ -0,0 +1,44 @@
+; It is assumed that the test will not fail
+
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+
```

Comment:
Please add checks to demonstrate the result of the pass

Reply:
Done